climada_petals/hazard/rf_glofas/cds_glofas_downloader.py

```diff
@@ -0,0 +1,286 @@
+"""
+This file is part of CLIMADA.
+
+Copyright (C) 2017 ETH Zurich, CLIMADA contributors listed in AUTHORS.
+
+CLIMADA is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free
+Software Foundation, version 3.
+
+CLIMADA is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along
+with CLIMADA. If not, see <https://www.gnu.org/licenses/>.
+
+---
+
+Functions for downloading GloFAS river discharge data from the Copernicus Climate Data
+Store (CDS).
+"""
```

Comment:
TODO: Update docs in this file

Reply:
I updated the docstrings of the functions to now include the `use_case` parameter as well as all other missing parameters.